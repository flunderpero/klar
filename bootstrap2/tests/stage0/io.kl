fn main():
    let c = getchar()
    while c > 0:
        print(f"{c}")
        c = getchar()
    end
end
