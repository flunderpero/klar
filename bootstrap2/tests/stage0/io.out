115
116
100
105
110
95
116
101
115
116
95
105
110
112
117
116
10
